src/index.html

```diff
@@ -5,9 +5,41 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page">
+    <div class="card" data-qa="card">
+      <img class="card__image"
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 image">
+
+      <h1 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__stars">
+        <div class="stars stars--n">
```

Comment:
What does `--n` means?
it should be a modifier (1-5) and show direct number of stars

Reply:
It was no really clear here. I put '--n' (where 'n' is the number of stars) not '--4', because the task indicated that this number might change. That's why I created '$stars-count: 4' variable and decided to keep the modifier neutral in case the number of stars will be changed.